optaplanner-core/src/main/java/org/optaplanner/core/impl/heuristic/selector/move/generic/PillarChangeMove.java

```diff
@@ -125,28 +123,22 @@ public String getSimpleMoveTypeDescription() {
     }
 
     @Override
-    public boolean equals(Object o) {
+    public boolean equals(final Object o) {
```

Comment:
(suggestion, non-blocking) Code style: normally we don't put "final" keywords in method parameters.